reader_test.go

```diff
@@ -529,50 +471,36 @@ func checkIpv6(c *C, reader *Reader) {
 		data := map[string]string{"ip": valueAddress}
 		var result map[string]string
 		err := reader.Lookup(net.ParseIP(keyAddress), &result)
-		if err != nil {
-			c.Logf("unexpected error while doing lookup: %v", err)
-			c.Fail()
-		}
-		c.Assert(result, DeepEquals, data)
+		assert.Nil(t, err, "unexpected error while doing lookup: %v", err)
+		assert.True(t, assert.ObjectsAreEqual(result, data))
 	}
 
 	for _, address := range []string{"1.1.1.33", "255.254.253.123", "89fa::"} {
 		var result map[string]string
 		err := reader.Lookup(net.ParseIP(address), &result)
-		if err != nil {
-			c.Logf("unexpected error while doing lookup: %v", err)
-			c.Fail()
-		}
-		c.Assert(result, IsNil)
+		assert.Nil(t, err, "unexpected error while doing lookup: %v", err)
+		assert.Nil(t, result)
 	}
 }
 
 func BenchmarkMaxMindDB(b *testing.B) {
-	db, err := Open("GeoLite2-City.mmdb")
-	if err != nil {
-		b.Fatal(err)
-	}
+	db, err := Open("test-data/test-data/GeoIP2-City-Test.mmdb")
```

Comment:
The benchmarks require that you have a local copy of `GeoLite2-City.mmdb`, which I don't check in due to its size. I should probably mention this in the docs. Benchmarking the tests databases wouldn't be very representative as they have almost no records.